web/src/views/DandisetLandingView/DandisetPublish.vue

```diff
@@ -129,7 +129,14 @@
                     class="py-0"
                   >
                     <div
-                      v-if="currentDandiset"
+                      v-if="currentDandiset && currentDandiset.status === 'Pending'"
+                      class="text-caption"
+                    >
+                      Validation is pending.
+                    </div>
+
+                    <div
+                      v-else-if="currentDandiset && currentDandiset.status === 'Invalid'"
```

Comment:
```suggestion
                      v-else-if="currentDandiset && currentDandiset.status === 'Invalid'" && currentDandiset.version_validation_errors.length
```
since I think it would be marked invalid also if there are asset errors. So we end up with smth like:

![image](https://user-images.githubusercontent.com/39889/167901930-91a7ecec-6278-4a14-a455-abf99a1b5725.png)